libs/SmartSync/SmartSync/Classes/Util/SFSyncDownTarget.m

```diff
@@ -43,7 +43,7 @@ @implementation SFSyncDownTarget
 #pragma mark - From/to dictionary
 
 + (SFSyncDownTarget*) newFromDict:(NSDictionary*)dict {
-    // We should have an implementation class unless sync down was created with SDK before 5.1 and is not custom
+    // We should have an implementation class or a target type
```

Comment:
Comment was not quite accurate - hybrid apps might not specify implementation class